tox.ini

```diff
@@ -36,6 +37,12 @@ commands =
         --diff \
         reviewrot/ test/
 
+[testenv:black-format]
+skip_install = true
+deps =
+    black
+commands =
+    black reviewrot/ test/
```

Comment:
So for context, this project previously didn't have any `flake8` or `black` linting. 
I wrote a previous patch to PEP8 bomb the project, and added these linters to Tox.

My concern here is how this `black-format` would auto format, and I'm not 100% sure that's the best approach. 

I'd rather see folks use Tox locally, prior to pushing, and amend their commits if necessary. That way all code changes are tracked by a commit end-to-end, and not modified in-flight. Besides that, I also wonder what happens when folks GPG sign commits that get modified by a workflow, does the attestation get lost in translation?

Reply:
`tox -e black-format` is only meant to be run manually (not triggered with `tox`). It is just a shortcut to the specific `black` command so you do not need to remember it (or install the utility).

`tox -e black` should run automatically and fail if reformat is needed (we use this approach in other internal repos too).